lib/_emerge/JobStatusDisplay.py

```diff
@@ -79,6 +75,13 @@ def _set_width(self, width):
         object.__setattr__(self, "width", width)
         object.__setattr__(self, "_jobs_column_width", width - 32)
 
+    def sigwinch(self):
+        if self._isatty:
```

Comment:
I guess we could also simply do nothing, i.e., early exit,  if `not self._isatty()`.

```python
def sigwinch(self):
  if not self._isatty:
    return
  width = portage.output.get_term_size()[1]
  self._set_width(width)
```

If that is possible, then it would be my personal preference. But nothing I would insist on.

Reply:
updated to use your version.